net-misc/frr/frr-7.5.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Gentoo Authors
+# Copyright 2020-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
1. about `py3.9`?
2. revbump?
3. `src_compile` is terrible

Reply:
Done; done; done